app/main.js

```diff
@@ -409,7 +409,7 @@ function startMicrobreak () {
       frame: false,
       show: false,
       transparent: settings.get('transparentMode'),
-      backgroundColor: calculateBackgroundColor(),
+      opacity: settings.get('transparentMode') ? settings.get('opacity') : 1,
```

Comment:
We can't do this as this is not multiplatform supported. That's why we used more complicated way with `calculateBackgroundColor()`

Reply:
ok